docs/contributing.md

```diff
@@ -280,37 +331,39 @@ Alternatively, you can run
 git push
 ```
 
-This will provide the code to create a new branch in your GitHub repository. Copy the command and paste it into your terminal and press enter to run it.
+## 3.11 Creating Pull Requests
 
-##### **ii. If there are conflicting changes in the upstream repository**
+### 3.11.1 Push All Changes in your Issue Branch
 
-When you check the upstream repository, you may see output like this:
+Once you are satisfied with your changes, push them to the feature branch you made within your remote repository.
 
 ```bash
-remote: Enumerating objects: 184, done.
-remote: Counting objects: 100% (184/184), done.
-remote: Compressing objects: 100% (62/62), done.
-remote: Total 184 (delta 123), reused 162 (delta 122), pack-reused 0
-Receiving objects: 100% (184/184), 26.75 KiB | 6.69 MiB/s, done.
-Resolving deltas: 100% (123/123), completed with 43 local objects.
-From https://github.com/hackforla/peopledepot
- * [new branch]      develop    -> upstream/develop
-   56e6dc7..923215d  main       -> upstream/main
-
+git push --set-upstream origin <name-of-branch>
 ```
 
+### 3.11.2 Complete pull request from Github
```

Comment:
`### 3.11.2 Complete Pull Request From Github`

Reply:
Done.  Left from lowercase as it is a preposition :-).